common/core/web/utils/package.json

```diff
@@ -5,7 +5,7 @@
   "main": "./dist/index.js",
   "types:": "./dist/index.d.ts",
   "scripts": {
-    "build": "tsc -p ./src/tsconfig.json",
+    "build": "node ../../../../resources/gosh.js ./build.sh -skip-dependency-install",
```

Comment:
Are we assuming that this will always be called by other packages, and never built directly itself?  The build will fail if `node_modules` isn't in place.

Reply:
Okay, removing `-skip-dependency-install`